lib/mesos/util.py

```diff
@@ -78,6 +78,18 @@ def format_commands_help(cmds):
 
     return help_string
 
+def format_flags_help(flags):
```

Comment:
I wouldn't pull this into a separate function, but rather embed it in the format_subcommands help